Donut_Shop.js

```diff
@@ -1,52 +1,81 @@
-      for(var i = 0; i <= this.myArray.length; i++) {
-        var td = document.createElement('td');
-        td.textContent = this.myArray[i];
-        elRows.appendChild(td);
-      }
-      td.textContent = dailyDonuts;
-    elRows.appendChild(td);
+    TopPotDonuts.prototype.render = function() {
+      // 1. Info needed from constructor obj: location name, hourly totals, daily totals.
+      //var dailyDonuts = this.total; //Grabbing this.total value and storing in dailyDonuts
+      //var hourlyDonuts = this.hourlyArray; //stores results of daily total method
+      //var locationName = this.franchise; //grabs location names
+      // 2. Create row elemnt in which to push new data from user inputs
+      console.log(this.total);
+      var row = document.createElement('tr');
+      // 3. Create data element for row
+      var data = document.createElement('td');
+      // 4. Put data from user into td tags
+      data.textContent = this.franchise;
+      //5. Append newly created data to tr tag
+      row.appendChild(data);
+      // 6. Create 'for' loop to iterate through hourlyArray and put in table
+      for(var i = 0; i < this.hourlyArray.length; i++) {
+        data = document.createElement('td');
+      data.textContent = this.hourlyArray[i];
+      row.appendChild(data);
+      };
+       // 3. Create data element for row
+      data = document.createElement('td');
+      // 4. Put data from user into td tags
+      data.textContent = this.total;
+      row.appendChild(data);
 
+      //Take completed row and inster into table body
+      document.getElementById('table-body').appendChild(row);
     };
-    var downtown = new topPotDonuts('downtown', {minCustomersPerHour: 8, maxCustomersPerHour: 43, avrgDonutsPerCust: 4.5});
-    var capitolHill = new topPotDonuts('capitolHill', {minCustomersPerHour: 4, maxCustomersPerHour: 47, avrgDonutsPerCust: 2});
-    var southLakeUnion = new topPotDonuts('southLakeUnion', {minCustomersPerHour: 9, maxCustomersPerHour: 23, avrgDonutsPerCust: 6.33});
-    var wedgewood = new topPotDonuts('wedgewood', {minCustomersPerHour: 2, maxCustomersPerHour: 28, avrgDonutsPerCust: 1.25});
-    var ballard = new topPotDonuts('ballard', {minCustomersPerHour: 8, maxCustomersPerHour: 58, avrgDonutsPerCust: 3.75});
 
+
+    var downtown = new TopPotDonuts('downtown', {minCustomersPerHour: 8, maxCustomersPerHour: 43, avrgDonutsPerCust: 4.5});
+    var capitolHill = new TopPotDonuts('capitolHill', {minCustomersPerHour: 4, maxCustomersPerHour: 47, avrgDonutsPerCust: 2});
+    var southLakeUnion = new TopPotDonuts('southLakeUnion', {minCustomersPerHour: 9, maxCustomersPerHour: 23, avrgDonutsPerCust: 6.33});
+    var wedgewood = new TopPotDonuts('wedgewood', {minCustomersPerHour: 2, maxCustomersPerHour: 28, avrgDonutsPerCust: 1.25});
+    var ballard = new TopPotDonuts('ballard', {minCustomersPerHour: 8, maxCustomersPerHour: 58, avrgDonutsPerCust: 3.75});
+
+    downtown.dailyTotal();
```

Comment:
You might consider how you could refactor the '.dailyTotal()' and '.render()' calls into for-loops, which would consolidate your code and provide you with some level of simplicity and scalability for any future models. 

You don't have to do the work, but giving thought to how that's done (Tyler demoed it yesterday) will help with your future code.